Src/DependencyCollector/Shared/Implementation/ProfilerHttpProcessing.cs

```diff
@@ -323,9 +322,12 @@ private object OnBegin(object thisObj, bool isAsyncCall)
 
                     // Add the parent ID
                     var parentId = telemetry.Id;
-                    if (!string.IsNullOrEmpty(parentId) && webRequest.Headers[RequestResponseHeaders.StandardParentIdHeader] == null)
+                    if (!string.IsNullOrEmpty(parentId))
                     {
-                        webRequest.Headers.Add(RequestResponseHeaders.StandardParentIdHeader, parentId);
+                        if (webRequest.Headers[RequestResponseHeaders.StandardParentIdHeader] == null)
```

Comment:
what if downstream service has lower version of AppInsights? If we care about backward compatibility, we should have both